TODO

```diff
@@ -1,10 +1,7 @@
-provider TODO -
-  - password should be a property and not a param
-     - what if we tried to log in as that user?
+provider TODO:
   - can permissions from list_user_permissions contain whitespace?
-  - what about defaultfor :true?
-  - prefetching for performance
+	- https://github.com/puppetlabs/puppetlabs-rabbitmq/blob/master/lib/puppet/type/rabbitmq_user_permissions.rb#L54 Should I go ahead and disallow whitespace?
+  - what about default for :true?
+	- What does this mean?
```

Comment:
Puppet doesn't have literal `true` and `false` values; it uses symbols `:true` and `:false` instead.